Automattic/jetpack
# Boost: Fix Overall Score Typo

Fixes #39625

## Proposed changes:

* Text-based fix for typo raised in #39625 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Go to "Jetpack > Boost"
* Click on the tooltip and see that the typo is fixed ("sites" is now "site's"):

![Screenshot on 2024-10-30 at 01-47-38](https://github.com/user-attachments/assets/76bc57bf-7699-4eb9-b811-a15652ab74b8)
